static/usr/lib/systemd/system/plymouth-start.service.d/core-override.conf

```diff
@@ -0,0 +1,9 @@
+[Unit]
+After=snap-initramfs-mounts.service
+Requires=plymouth-quit.service splash-client.service
+Requires=plymouth-show-run.service plymouth-show-install.service
+Requires=plymouth-show-recover.service plymouth-show-factory-reset.service
```

Comment:
I would prefer `Wants`. Specially if we move `plymouth-quit.service` from `multi-user.target.wants`.

Reply:
Done